Makefile

```diff
@@ -1,5 +1,14 @@
-run:
-	go run ./examples/host
+.DEFAULT_GOAL := build
+CHARMIL_BIN_FILE = charmil

```

Comment:
charmil CLI does what?

Reply:
This is outdated code. 
Remember we were once planning to include a Charmil CLI which SDK users can interact with? This was meant for that.

But I've removed it now, since there have been changes in our plans and we're no longer going to implement this